Fabric/build.gradle

```diff
@@ -1,21 +1,25 @@
 plugins {
-    id 'fabric-loom' version '1.0-SNAPSHOT'
-    id 'maven-publish'
+    id 'java'
     id 'idea'
+	id 'fabric-loom' version '1.1.+'
```

Comment:
Fabric and Quilt looms conflict when added via root project. Other plugins miss the version specification in subprojects because their versions are defined in root project.